cas/service.c

```diff
@@ -1132,7 +1133,13 @@ static int cas_dtm0_logrec_add(struct m0_fom *fom0,
 			break;
 		}
 	}
-	rc = m0_xcode_obj_enc_to_buf(&M0_XCODE_OBJ(m0_cas_op_xc, cas_op(fom0)),
+	dtm_payload.cdg_cas_op = *cas_op(fom0);
+	dtm_payload.cdg_cas_opcode = m0_fop_opcode(fom0->fo_fop);
+	if (dtm_payload.cdg_cas_opcode == M0_CAS_DEL_FOP_OPCODE) {
+		dtm_payload.cdg_cas_op.cg_rec.cr_rec->cr_val.ab_type = M0_RPC_AT_EMPTY;
```

Comment:
[Suggestion] To make it more readable,

struct m0_cas_rec *rec = dtm_payload.cdg_cas_op.cg_rec.cr_rec;

r_rec->cr_val.ab_type = M0_RPC_AT_EMPTY;
cr_rec->cr_val.u.ab_buf = M0_BUF_INIT0;

Reply:
done